### PR TITLE
Add Rubocop

### PR DESCRIPTION
**Pull request by benbalter**
Reviewing https://github.com/jekyll/jemoji/pull/58, I noticed the Gem didn't have Rubocop. 

This fixes that and gets tests passing.

### CODE REVIEW

File: spec/jemoji_spec.rb [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
     }))
   end
   let(:emoji)       { described_class }
   let(:site)        { Jekyll::Site.new(configs) }
   let(:default_src) { "https://assets-cdn.github.com/images/icons/" }
-  let(:result)      { "<img class=\"emoji\" title=\":+1:\" alt=\":+1:\" src=\"#{default_src}emoji/unicode/1f44d.png\" height=\"20\" width=\"20\" align=\"absmiddle\">" }
+  let(:result) do
+    result = "<img class=\"emoji\" title=\":+1:\" alt=\":+1:\" "
+    result << "src=\"#{default_src}emoji/unicode/1f44d.png\" "
+    result << "height=\"20\" width=\"20\" align=\"absmiddle\">"
```
Comment (parkr) [inline spec/jemoji_spec.rb]: Lol we should totally use heredocs for this...

**Comment by parkr** [comment]
@benbalter feel free to ship this once CI is passing!

File: spec/jemoji_spec.rb [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
   end
   let(:emoji)       { described_class }
   let(:site)        { Jekyll::Site.new(configs) }
   let(:default_src) { "https://assets-cdn.github.com/images/icons/" }
-  let(:result)      { "<img class=\"emoji\" title=\":+1:\" alt=\":+1:\" src=\"#{default_src}emoji/unicode/1f44d.png\" height=\"20\" width=\"20\">" }
+  let(:result) do
+    result = "<img class=\"emoji\" title=\":+1:\" alt=\":+1:\" "
+    result << "src=\"#{default_src}emoji/unicode/1f44d.png\" "
+    result << "height=\"20\" width=\"20\">"
+  end
```
Comment (ashmaroli) [inline spec/jemoji_spec.rb]: You'll need to change this again once you start enforcing frozen strings..

**Comment by parkr** [comment]
@jekyllbot: merge

**Pull request merged**